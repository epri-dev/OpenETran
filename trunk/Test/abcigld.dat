1 3 30.0 1 1 0.003e-6 20.0e-6

cable 1 400.0 3.0e8 0.0e3

surge 20.0e3 1.0e-6 20.0e-6 0.0e-6
pairs 1 0
poles 2

arrbez 0.0e3 40.0e3 0.051 1.0e-6 0.3 1
pairs 1 0
poles 2

meter
pairs 1 0
poles 2


