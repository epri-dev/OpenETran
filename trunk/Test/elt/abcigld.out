  N   span     dT   Tmax
  1  30.00 3e-009 2e-005
Z-phase
  4.00000e+002
Z-modal
  4.00000e+002
Modal Transformation (Ti)
  1.00000e+000
Power-frequency source at pole 1.
   #        vp        vm         i
   1       0.0       0.0       0.0
Power-frequency source at pole 3.
   #        vp        vm         i
   1       0.0       0.0       0.0
Meter at pole 2, from 1 to 0, max V   = 4.838572e+004
Meter at pole 2, arrester max I       = 1.977629e+004
nr_iter = 29881, nr_max = 11
Execution time [sec]: 0.062
